include/vkgcDefs.h

```diff
@@ -846,10 +846,10 @@ class ICache {
   ///
   /// Valid handles are always non-null.
   ///
-  /// \param hash the hash key for the cache entry
-  /// \param allocateOnMiss if true, a new cache entry will be allocated when none is found
-  /// \param pHandle will be filled with a handle to the cache entry on Success, NotReady and allocateOnMiss if NotFound
-  /// \return success code, possible values:
+  /// @param hash the hash key for the cache entry
```

Comment:
If your script can correct the style to `// @param hash : The hash key for the cache entry`, that is very good.
1. Add ":" as delimitor
2. Comment is uppercase-initial after ":"

This is the same for @return. The style could be added to codingStandard.md

`// @return : Return vale`

Also, take this chance, I would like to change /// to // for all files and make them consistent

Reply:
Done